public/js/clients/index.js

```diff
@@ -13,7 +13,23 @@ function getClient() {
   return firefox.clientCommands;
 }
 
-function startDebugging(targetEnv, tabId, actions) {
+function startDebugging(connTarget, actions) {
+  if (connTarget.type === "node") {
+    return startDebuggingNode(connTarget.param, actions);
+  }
+
+  const target = connTarget.type === "chrome" ? chrome : firefox;
+  return startDebuggingTab(target, connTarget.param, actions);
+}
+
+function startDebuggingNode(url, actions) {
+  clientType = "chrome";
+  return chrome.connectNode(`ws://${url}`).then(() => {
```

Comment:
I don't love the name `connectNode`. This isn't specific to node at all, it's just passing the websocket URL directly. It could be anything. People may use this API to build a different toolbox, or we might eventually have a websocket from Firefox to pass in. I'd rather it be something like `connectWebsocket`.